packages/devtools/src/launcher.js

```diff
@@ -58,6 +57,7 @@ async function launchChrome (capabilities) {
 }
 
 function launchFirefox (capabilities) {
+    const puppeteerFirefox = require('puppeteer-firefox')
```

Comment:
To fix unit tests, please modify the [`puppeteer-firefox`](https://github.com/webdriverio/webdriverio/blob/master/packages/devtools/tests/__mocks__/puppeteer-firefox.js) mock to use CommonJS syntax:

```js
module.exports = {
    launch: jest.fn()
}
```

Reply:
Awe ok will do!